rtloader/test/datadog_agent/datadog_agent.go

```diff
@@ -356,3 +358,8 @@ func obfuscateMongoDBString(cmd *C.char, errResult **C.char) *C.char {
 		return nil
 	}
 }
+
+//export emitAgentTelemetry
+func emitAgentTelemetry(check *C.char, metric *C.char, value C.float, metricType *C.char) {
+	// NOOP
```

Comment:
Since there was a bug in the argument parsing, would it be possible to verify that we get the expected arguments here?